saw-core/src/Verifier/SAW/SharedTerm.hs

```diff
@@ -414,6 +366,8 @@ scResolveUnambiguous sc nm =
        do nms <- mapM (scFindBestName sc . snd) xs
           fail $ unlines (("Ambiguous name " ++ show nm ++ " might refer to any of the following:") : map show nms)
 
+-- TODO: This differs from 'Verifier.SAW.Name.bestAlias' in the
+-- 'ModuleIdentifier' case. Should we redefine this to match?
```

Comment:
I think it's probably better to redefine in terms of `bestAlias`, which will try to use an unqualified name first, and is probably what we actually want.

Reply:
Done in acf9c406330b89fa8568682b809bb9974d26869b.